src/main/java/com/google/devtools/build/lib/skyframe/RepoFileFunction.java

```diff
@@ -140,7 +136,7 @@ private static String getDisplayNameForRepo(
     return displayName;
   }
 
-  private PackageArgs evalRepoFile(
+  private RepoFileValue evalRepoFile(
       StarlarkFile starlarkFile,
       RepositoryName repoName,
       RepositoryMapping repoMapping,
```

Comment:
remove the unused repo mapping args

Reply:
They are not used (see line 161, the `LabelConverter` call)

Reply:
assuming you meant "they are not _unused_", they should be now.

Reply:
ping -- this arg should be removed